pages/task/read/views/models/project.jsx

```diff
@@ -17,19 +17,31 @@ import Deadline from '../components/deadline';
 import { fields } from '../../../../project/update-licence-holder/schema/experience-fields';
 import { schema as projectSchema } from '../../../../project/schema';
 
-const selector = ({ static: { project, establishment, version } }) => ({ project, establishment, version });
+const selector = ({ static: { project, establishment, version, values } }) => ({ project, establishment, version, values });
 
 // declarations can be 'Yes', 'No', or 'Not yet'
 const declarationConfirmed = declaration => declaration && declaration.toLowerCase() === 'yes';
 
-function EstablishmentDiff({ task }) {
+function EstablishmentDiff({ task, isComplete }) {
```

Comment:
Completeness can be trivially determined from the task, and is entirely linked to the task, so I wouldn't pass it as a separate prop.